### DISCUSSION TITLE
I can't run the 0.4editor

### DISCUSSION THREAD

**Discussion by shengkongdeng** [Adrianotiger/desktopPet#63] [Q&A]
My English level is limited，So I used a translator to write this article。
I run the 0.4 editor on my 64 bit win10 computer，But his interface doesn't work on my desktop。
So I want to get some help now。
It looks like this
![图片](https://user-images.githubusercontent.com/74752988/111114260-f4e5e980-859d-11eb-8191-7fcc534faeda.png)

**Comment by Adrianotiger** [comment]
Hi,
It is a simple .net application. So you should see it without any problems on win10.

If it doesn't start, it could be the antivirus or a windows policy.

Do you extract all files from the zip? There are 3-4 libraries (dll) used by the application. They need to be in the same folder.

**Reply by shengkongdeng** [reply]
I tried to turn off antivirus software and also determine the libraries file and application in the same location

But it didn't work.

I can find it in the background process.

It's just that its UI won't show up on my desktop.

And this program runs on my friend's computer without problem.

Should I configure something?

![图片](https://user-images.githubusercontent.com/74752988/111240872-03ccaa80-8637-11eb-87bb-839498141b61.png)

**Reply by shengkongdeng** [reply]
A new situation has emerged.
I hung in the program for a period of time after an error, it seems that the problem is about the UI DLL file.

I don't know much about the code

I posted the error message↓↓↓

************** 异常文本 **************
System.Exception: 被呼叫方(服务器[不是服务器应用程序])不可用并已消失。所有连接均无效。呼叫可能已执行。 (异常来自 HRESULT:0x80010007 (RPC_E_SERVER_DIED))
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.Toolkit.Win32.UI.Controls.Interop.WinRT.WebViewControlProcess.<CreateWebViewControlHostAsync>d__16.MoveNext()
--- 引发异常的上一位置中堆栈跟踪的末尾 ---
   在 System.Runtime.CompilerServices.TaskAwaiter.ThrowForNonSuccess(Task task)
   在 System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   在 Microsoft.Toolkit.Forms.UI.Controls.WebViewControlProcessExtensions.CreateWebViewControlHost(WebViewControlProcess process, IntPtr hostWindowHandle, Rectangle bounds)
   在 Microsoft.Toolkit.Forms.UI.Controls.WebView.Initialize()
   在 Microsoft.Toolkit.Forms.UI.Controls.WebView.System.ComponentModel.ISupportInitialize.EndInit()
   在 Microsoft.Toolkit.Forms.UI.Controls.WebViewCompatibilityAdapter.Initialize()
   在 Microsoft.Toolkit.Forms.UI.Controls.WebViewCompatible..ctor()
   在 PetEditor.WebForm.InitializeComponent()
   在 PetEditor.WebForm..ctor()
   在 PetEditor.MainWindow.MainWindow_Load(Object sender, EventArgs e)
   在 System.Windows.Forms.Form.OnLoad(EventArgs e)
   在 System.Windows.Forms.Form.OnCreateControl()
   在 System.Windows.Forms.Control.CreateControl(Boolean fIgnoreVisible)
   在 System.Windows.Forms.Control.CreateControl()
   在 System.Windows.Forms.Control.WmShowWindow(Message& m)
   在 System.Windows.Forms.Control.WndProc(Message& m)
   在 System.Windows.Forms.ScrollableControl.WndProc(Message& m)
   在 System.Windows.Forms.Form.WmShowWindow(Message& m)
   在 System.Windows.Forms.Form.WndProc(Message& m)
   在 System.Windows.Forms.Control.ControlNativeWindow.OnMessage(Message& m)
   在 System.Windows.Forms.Control.ControlNativeWindow.WndProc(Message& m)
   在 System.Windows.Forms.NativeWindow.Callback(IntPtr hWnd, Int32 msg, IntPtr wparam, IntPtr lparam)
